examples/platforms/posix/radio.c

```diff
@@ -762,6 +816,16 @@ void radioTransmit(struct RadioMessage *aMessage, const struct otRadioFrame *aFr
             exit(EXIT_FAILURE);
         }
     }
+#else  // OPENTHREAD_POSIX_VIRTUAL_TIME == 0
+    struct Event event;
+
+    event.mDelay      = 1; // 1us for now
+    event.mEvent      = OT_SIM_EVENT_RADIO_RECEIVED;
+    event.mDataLength = 1 + aFrame->mLength; // include channel in first byte
+    memcpy(event.mData, aMessage, event.mDataLength);
+
+    otSimSendEvent(&event);
+#endif // OPENTHREAD_POSIX_VIRTUAL_TIME == 0
```

Comment:
nit: OPENTHREAD_POSIX_VIRTUAL_TIME != 0

Reply:
I this this should be identical to the content after the corresponding `#if`. @jwhui

Reply:
Agree that the closing comment should be the same as the corresponding `#if`.